wallet/dapps-communication.md

```diff
@@ -182,18 +182,23 @@ required:
 ```yaml
 type: object
 properties:
-  balance:
+  finalBalance:
+    type: string
+    format: BigInt
+  currentBalance:
```

Comment:
It is named candidate_balance in core side

```suggestion
  candidateBalance:
```

Reply:
changed

Reply:
ok, changed